robottelo/constants/__init__.py

```diff
@@ -323,6 +323,15 @@
     'on_demand': "On Demand",
     'immediate': "Immediate",
 }
+DOWNLOAD_POLICY_ON_DEMAND = 'on_demand'
+DOWNLOAD_POLICY_IMMEDIATE = 'immediate'
+DOWNLOAD_POLICY_BACKGROUND = 'background'
```

Comment:
I am personally not a fan of this. I'd rather just see the values used in place of these constants. However, others are free to weigh in.